### DISCUSSION TITLE
Deleting lance index?

### DISCUSSION THREAD

**Discussion by cooperll** [lance-format/lance#1800] [Q&A]
Is there a standard way to delete an index after creating one with create_index? I don't see a simple way to do it currently.

**Comment by eddyxu** [comment]
It does not. Do you want to reclaim space ?

**Comment by muhajirdev** [comment]
Was looking for the same answer 👀

**Comment by wjones127** [comment]
You can delete them now with `ds.drop_index()`. https://lancedb.github.io/lance-python-doc/all-modules.html#lance.dataset.LanceDataset.drop_index
